### ISSUE TITLE
定制的文件系统插件引用DeepRec库抛undefined symbol

### ISSUE THREAD

**Issue by welsonzhang** [alibaba/DeepRec#518]
内部版本ps想使用DeepRec的推理能力，移植遇到undefined symbol异常。

参考https://tensorflow.juejin.im/extend/add_filesys.html 定义tensorflow的FileSystem ，引用DeepRec抛undefined symbol具体错误如下：

(1) 在load so的时候失败：
    lib_handle = py_tf.TF_LoadLibrary(library_filename)
tensorflow.python.framework.errors_impl.NotFoundError: 

(2) 抛undefined symbol异常：
undefined symbol: _ZN10tensorflow10FileSystem10FilesExistERKSt6vectorISsSaISsEEPS1_INS_6StatusESaIS6_EE

(3) 尝试strings查看对应的导出符号：
strings libtensorflow_framework.so.1 |grep tensorflow10FileSystem10FilesExistERKSt
_ZN10tensorflow10FileSystem10FilesExistERKSt6vectorISsSaISsEEPS1_INS_6StatusESaIS6_EE
_ZN10tensorflow10FileSystem10FilesExistERKSt6vectorISsSaISsEEPS1_INS_6StatusESaIS6_EE

(4) 最终定位出来是应为ABI不兼容导致，在编译DeepRec的时候，添加ABI=0的参数， 具体编译参数如下：
bazel build --cxxopt="-D_GLIBCXX_USE_CXX11_ABI=0" --host_cxxopt="-D_GLIBCXX_USE_CXX11_ABI=0" -c opt --config=opt --config=mkl_threadpool //tensorflow/tools/pip_package:build_pip_package

(5) 问题解决。

**Comment by liutongxuan** [comment]
good catch, TF use ABI=0 by default, but DeepRec need add ABI=0 explicitly.   We'll take care of this incompatibility.
